frontend/package.json

```diff
@@ -22,6 +22,7 @@
     "inter-ui": "^3.15.0",
     "lodash-es": "^4.17.21",
     "moment": "^2.29.1",
+    "naive-ui": "^2.23.1",
```

Comment:
Did you merge the main branch? The previous PR is adding naive-ui?

Reply:
I did merge into main... Maybe it is a bug at Github.
Anyway, I create a new branch from the main and pasted there, It should look good now.